bin/ck8s

```diff
@@ -153,5 +154,17 @@ diagnostics)
   shift
   "${here}/diagnostics.bash" "${@}"
   ;;
+version)
+  shift
+  case "${1:-both}" in
+  both)
+    log_info "sc version:" "$(get_version "sc")"
+    log_info "wc version:" "$(get_version "wc")"
+    ;;
+  sc | wc)
+    log_info "${1} version:" "$(get_version "${1}")"
+    ;;
+  esac
+  ;;
```

Comment:
Nice to have this command. I think it could be even better if it also printed the version in the config, so you can easily see if they are the same. Does that sound good to you as well?

Reply:
Sounds good.

Should just be one version in `defaults/common-config.yaml`, or are there cases where wc and sc may differ?

Reply:
I think that the only logical place to override the version is in common. It might be technically possible to have it in sc or wc, but that is not intended.

So just comparing with common should be fine, but doing the regular merge of sc with common and wc with common might be safest.

Reply:
`ck8s version` can now show version from config and from cluster(s)